assets/javascripts/kitten/components/form/select-with-state.js

```diff
@@ -118,6 +121,8 @@ SelectWithState.defaultProps = {
   multi: false,
   labelText: null,
   error: false,
+  valid: false,
+  disabled: false,
```

Comment:
Est-ce qu'on peut ajouter trois tests unitaires ?

1. Si on donne `valid` ça doit ajouter une classe `is-valid`
2. Si on donne `disabled` ça doit ajouter une classe `is-disabled`
3. Si on donne `disabled` ça passe `disabled` au `SelectWithMultiLevel`